miio/device.py

```diff
@@ -168,6 +168,13 @@ def _fetch_info(self) -> DeviceInfo:
                 "Unable to request miIO.info from the device"
             ) from ex
 
+    @property
+    def device_id(self):
```

Comment:
```suggestion
    def device_id(self) -> int:
```
As this is part of the base class, please also add a simple test to `test_device.py` that it functions (and will keep functioning in the future) with some example values.

Reply:
I think I have added a working test, could you check if this is what you ment?